api/internal/stats/index.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+define("ROOTDIR", isset($_POST["rootdir"]) ? $_POST["rootdir"] : "");
+define("REAL_ROOTDIR", "../../../");
+
+require_once REAL_ROOTDIR."src/initializer.php";
+use \Catalyst\API\Endpoint;
+
+Endpoint::init(true, Endpoint::AUTH_REQUIRE_NONE);
+
+header("Content-Type: image/svg+xml");
+
+$errorSvg = <<<ERR_SVG
```

Comment:
Really basic "ERROR" message for...idk what would cause this.  But it's here just in case anyone tries messing with the endpoint.